packages/shared/src/s3/commands.ts

```diff
@@ -22,9 +29,15 @@ export class S3Commands {
   }
 
   // Bucket command members
+  createBucket: CreateBucket = (input) =>
+    this.s3Client.send(new CreateBucketCommand(input));
+
   listBuckets: ListBuckets = (input) =>
     this.s3Client.send(new ListBucketsCommand(input));
 
+  updateBucketTags: UpdateBucketTags = (input) =>
```

Comment:
it will be easier to find, if we align the name of the member function with the actual S3 command:)
```suggestion
  putBucketTags: PutBucketTags = (input) =>
```

Reply:
or is there any specific reason for naming it differently ??

Reply:
I wanted to align to CRUD naming (hence `Update`). Are not we following it?

Reply:
idea was to keep name similar to it's corresponding S3 command, just camel case and removing unnecessary postfix like `Command`...
even class name is `S3Commands`...